openupgradelib/openupgrade.py

```diff
@@ -657,12 +657,14 @@ def rename_fields(env, field_spec, no_deep=False):
         # TODO: Rename when the field is part of a submodel (ex. m2one.field)
         cr.execute("""
             UPDATE ir_filters
-            SET domain = replace(domain, %(old_pattern)s, %(new_pattern)s)
+            SET domain = regexp_replace(
+                domain, %(old_pattern)s, %(new_pattern)s, 'g'
```

Comment:
Why this extra arg `g`?

Reply:
@pedrobaeza  As far as I know, the flag ‘g’ is used to indicate that replace all the occurrences that appear.
If no arg 'g' like this, it wont work in this case (i have muiltiple word which need to replace)
![Screenshot from 2022-07-15 09-40-54](https://user-images.githubusercontent.com/56789189/179136411-a976cb8c-f9f3-46e9-ac4f-b37449623a4c.png)

But with arg 'g', it work for me: 
![Screenshot from 2022-07-15 09-42-55](https://user-images.githubusercontent.com/56789189/179136594-82e85d6e-cce0-44c6-a8d8-90218db1331f.png)